src/Components/Home/HomeEditForm.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+//Forms
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+//Styles
+import Swal from 'sweetalert2';
+import {
+  Button,
+  Typography,
+  Container,
+  TextField,
+  Paper,
+  Stack,
+} from '@mui/material';
+import s from '../../Styles/HomeEditForm.module.css';
+//Components
+import SlidesForm from '../Slides/SlidesForm';
+
+const validationSchema = Yup.object({
+  welcomeText: Yup.string()
+    .min(20, 'El texto de bienvenida debe ser de al menos 20 caracteres')
+    .required('Este campo es obligatorio'),
+});
+
+export default function HomeFormEdit() {
+  const [welcomeText, setWelcomeText] = useState('');
+
+  const formik = useFormik({
+    initialValues: {
+      welcomeText: '',
+    },
+    validationSchema,
+    onSubmit: (values, { setSubmitting }) => {
+      Swal.fire({
+        text: 'El texto de bienvenida se actualizó correctamente!',
+        icon: 'success',
+        confirmButtonText: 'Hecho',
+      });
+      setWelcomeText(values.welcomeText);
+      setSubmitting(false);
+    },
```

Comment:
the function on onSubmit should be a separated function so it's easier to test and scale. Maybe you can call that function onFormikSubmit